homeassistant/components/template/fan.py

```diff
@@ -340,23 +350,23 @@ def _update_state(self, result):
             self._state = None
             return
 
-        # Validate state
-        if result in _VALID_STATES:
```

Comment:
Remove the `_VALID_STATES` constant if it's not needed anymore

Reply:
removing